internal/pkg/flink/internal/store/store_utils.go

```diff
@@ -233,43 +263,52 @@ func parseResetStatement(statement string) (string, error) {
 	}
 
 	if len(words) > 2 {
-		return "", &types.StatementError{Msg: "Error: too many keys for RESET provided. Usage example: RESET 'key'."}
+		return "", &types.StatementError{
+			Msg:   "too many keys for RESET provided",
+			Usage: []string{"RESET 'key'"},
+		}
 	}
 
 	isFirstWordReset := strings.ToUpper(words[0]) == config.ConfigOpReset
 	key := strings.ToLower(words[1])
 	if !isFirstWordReset {
-		return "", &types.StatementError{Msg: "Error: Invalid syntax for RESET. Usage example: RESET 'key'."}
+		return "", &types.StatementError{
+			Msg:   "Invalid syntax for RESET",
+			Usage: []string{"RESET 'key'"},
+		}
 	}
 
 	if !strings.HasPrefix(key, "'") || !strings.HasSuffix(key, "'") {
-		return "", &types.StatementError{Msg: "Error: Invalid syntax for RESET, key must be enclosed by single quotes ''. Usage example: RESET 'key'."}
+		return "", &types.StatementError{
+			Msg:   "Invalid syntax for RESET, key must be enclosed by single quotes ''",
+			Usage: []string{"RESET 'key'"},
+		}
 	}
 
 	return strings.ReplaceAll(key, "'", ""), nil
 }
 
 func processHttpErrors(resp *http.Response, err error) error {
 	if err != nil {
-		return &types.StatementError{Msg: "Error: " + err.Error()}
+		return &types.StatementError{Msg: err.Error()}
 	}
 
 	if resp != nil && resp.StatusCode >= 400 {
 		if resp.StatusCode == http.StatusUnauthorized {
-			return &types.StatementError{Msg: "Error: Unauthorized. Please consider running confluent login again.", HttpResponseCode: resp.StatusCode}
+			return &types.StatementError{Msg: "Unauthorized. Please consider running confluent login again", HttpResponseCode: resp.StatusCode}
```

Comment:
```suggestion
			return &types.StatementError{
				Msg: "Unauthorized. Please run `confluent login`",
				HttpResponseCode: resp.StatusCode,
			}
```

Reply:
The second sentence here is more of a suggestion than an error. Should we introduce another field called "Suggestion"? This is what other CLI commands do.

Reply:
Added Suggestion field!